### PR TITLE
update Fundamentals page content

### PR DESCRIPTION
**Pull request by AlekseyKorzik**

### CODE REVIEW

File: docs/docs/setting-up/networking-instructions/fundamentals.md [hunk 1]
```diff
@@ -11,31 +11,44 @@ Bacalhau uses [libp2p](https://libp2p.io/) under the hood to communicate with ot
 
 Because bacalhau is built using libp2p, the concept of peer identity is used to identify nodes on the network.
 
-When you start a bacalhau node using `bacalhau serve`, it will look for an RSA private key in the `~/.bacalhau` directory. If it doesn't find one, it will generate a new one and save it there.
+Bacalhau uses 2 keys to identify the node and the user:
+- A private Ed25519 `libp2p_private_key` key is used to get the Peer ID in libp2p
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/fundamentals.md:15]: .. and identifies a server node.

File: docs/docs/setting-up/networking-instructions/fundamentals.md [hunk 2]
```diff
@@ -11,31 +11,44 @@ Bacalhau uses [libp2p](https://libp2p.io/) under the hood to communicate with ot
...
+- A private RSA key in the PKCS1 format `user_id.pem` is used to uniquely identify user on the network when making reqeusts from CLI
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/fundamentals.md:16]:
```suggestion
- A private RSA key in the PKCS1 format `user_id.pem` is used to uniquely identify the user on the network when making requests from CLI
```

File: docs/docs/setting-up/networking-instructions/fundamentals.md [hunk 3]
```diff
@@ -11,31 +11,44 @@ Bacalhau uses [libp2p](https://libp2p.io/) under the hood to communicate with ot
...
 
-You can override the directory where the private key is stored using the `BACALHAU_PATH` environment variable.
+By default, Bacalhau creates its own keys on installation. These keys can be found at the path given by the config:
+- key `User.KeyPath` for a RSA key, which by default is `~/.bacalhau/user_id.pem`
+- key `User.Libp2pkeypath` for an Ed25519 key, which by default is `~/.bacalhau/libp2p_private_key`
 
-Private keys are named after the port used for the libp2p connection which defaults to `1235`. By default when first starting a node, the private key will be stored in `~/.bacalhau/private_key.1235`.
+
+When you start a bacalhau node using `bacalhau serve`, it will look for these keys in the paths, specified in the environmental variable or config. If it doesn't find them, it will generate a new ones and save them there.
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/fundamentals.md:23]: the server command doesn't need the user_id.pem, and so really it's just looking for the libp2p_private_key, creating it if necessary.

File: docs/docs/setting-up/networking-instructions/fundamentals.md [hunk 4]
```diff
@@ -62,9 +75,13 @@ bacalhau serve \
 
 To ensure that the node can communicate with other nodes on the network, make sure the swarm port is open and accessible by other nodes.
 
+:::info
+In the libp2p the term swarm has been replaced by [switch](https://docs.libp2p.io/concepts/appendix/glossary/#switch), however swarm is still used for historical reasons
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/fundamentals.md:79]:
```suggestion
In libp2p the term swarm has been replaced by [switch](https://docs.libp2p.io/concepts/appendix/glossary/#switch), however swarm is still used for historical reasons 
```

Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/fundamentals.md]: fixed

File: docs/docs/setting-up/networking-instructions/fundamentals.md (hunk 3 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/fundamentals.md]: fixed

File: docs/docs/setting-up/networking-instructions/fundamentals.md (hunk 2 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/fundamentals.md]: fixed

File: docs/docs/setting-up/networking-instructions/fundamentals.md (hunk 1 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/fundamentals.md]: fixed

**Review by rossjones — APPROVED** [review]

**Pull request merged**